flyteadmin/pkg/repositories/transformers/execution.go

```diff
@@ -76,7 +75,7 @@ func CreateExecutionModel(input CreateExecutionModelInput) (*models.Execution, e
 	}
 	createdAt := timestamppb.New(input.CreatedAt)
 	closure := admin.ExecutionClosure{
-		Phase:         input.Phase,
+		Phase:         core.WorkflowExecution_UNDEFINED,
```

Comment:
nit, no need to explicitly set this, this is the default enum value

Reply:
good catch